docs/learn/learn-staking-advanced.md

```diff
@@ -381,20 +396,23 @@ already maxed out.
 
 {{ polkadot: DOT is an inflationary token. In fact, there is no maximum number of DOT. Inflation is designed
 to be approximately 10% annually, with validator rewards being a function of the amount staked
-and the remainder going to the treasury. DOT went through [redenomination](../general/redenomination.md) 
+and the remainder going to the treasury. DOT went through [redenomination](../general/redenomination.md)
 in 2020 that saw the DOT token supply increase by 100 times. The current token supply on Polkadot is <RPC network="polkadot" path="query.balances.totalIssuance" defaultValue={12230666300429914781} filter="humanReadable"/> (Over 1.2 Billion DOT). :polkadot }}
 
 {{ kusama: KSM is inflationary; there is no maximum number of KSM. Inflation is designed
 to be approximately slightly over 7.5% annually, with validator rewards being a function of the amount staked
-and the remainder going to the treasury. The current token supply on Kusama is 
-{{ kusama: <RPC network="kusama" path="query.balances.totalIssuance" defaultValue={12619256191792480093}/> :kusama }} (Over 12 Million KSM). :kusama }}
-
-There is an _ideal staking rate_ that the network tries to maintain. The goal is to have the *system staking rate* meet the *ideal staking rate*. The system staking rate would be the total amount staked over the total token supply, where the
-total amount staked is the stake of all validators and nominators on the network. The ideal staking
-rate accounts for having sufficient backing of {{ polkadot: DOT :polkadot }}
-{{ kusama: KSM :kusama }} to prevent the possible compromise of security while keeping the native
-token liquid. {{ polkadot: DOT :polkadot }}{{ kusama: KSM :kusama }} is inflated according to the 
-system staking rate of the entire network.
+and the remainder going to the treasury. The current token supply on Kusama is
+{{ kusama: <RPC network="kusama" path="query.balances.totalIssuance" defaultValue={12619256191792480093}/> :kusama }}
```

Comment:
```suggestion
and the remainder going to the treasury. The current token supply on Kusama is <RPC network="kusama" path="query.balances.totalIssuance" defaultValue={12619256191792480093}/>
```

This entire paragraph is already wrapped in conditonal rendering so you don't need to wrap the RPC component by itself here as it will actually mess up the formatting.

Before:
![image](https://user-images.githubusercontent.com/13341935/196609550-eea87d5e-f6b1-4910-896f-1a61bc210c50.png)

After:
![image](https://user-images.githubusercontent.com/13341935/196609651-5f7c45af-0cda-4eaf-a112-ab7b9d1d61ec.png)